src/sentry/tasks/store.py

```diff
@@ -685,8 +685,6 @@ def create_failed_event(
             data=issue["data"],
         )
 
-    event_processing_store.delete_by_key(cache_key)
```

Comment:
After creating a processing issue the event does not go to post-processing, so this should not be necessary

Reply:
Thanks. Restored now.